app/Helpers/Seconds To Human/Helper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+function secondsToHumanReadable(int $duration): string
+{
+    $days = floor($duration / (24 * 3600));
+    $hours = floor(($duration % (24 * 3600)) / 3600);
+    $minutes = floor(($duration % 3600) / 60);
+     
+
+    // Burada max day min minute olaraq deyisdirdim
+    $output = '';
+     
+    if ($days > 0) {
+        $output .= $days . ' day ';
+    }
+
+    if ($hours > 0) {
+        $output .= $hours . ' hrs ';
+    }
+    /*
+     Burda deyirsiz 2hr yerine 2hrs olmalidi, tam olaraq burani basa dusmedim.

```

Comment:
@Karz4 ingilis dilinde eger say 1-den boyuk olarsa o zaman `unit`in sonuna cem shekilcisi `s` artirilir. Meselen, 1 hour, 2 hours, 3 hours, 1 minute, 2 minutes, ve s. Buna gore de eger $days, $hours ve/ve ya $minutes 1-den boyuk olarsa, o zaman sonlarinda `s` olmalidi. Yeni, `hr` yox `hrs`, `min` yox `mins`, `day` yox `days.